avionics/common/include/radio.h

```diff
@@ -69,6 +69,8 @@ class Radio {
     constexpr static Hal::ms WATCHDOG_SEND_INTERVAL{3000};
     constexpr static fwd_cmd_t STOP_PARSE_FLAG = 1 << 0;
     constexpr static fwd_cmd_t CAN_SEND_FLAG = 1 << 1;
+    
+    static bool m_can_send; // Is XBee available to send
```

Comment:
On FLARE we seem to have standardized around using a trailing _ to indicate member vars e.g. can_send_

Reply:
Good catch!